client/me/concierge/book/info-step.js

```diff
@@ -48,7 +48,10 @@ class InfoStep extends Component {
 	};
 
 	setTimezone = timezone => {
-		this.props.updateConciergeSignupForm( { ...this.props.signupForm, timezone } );
+		this.props.updateConciergeSignupForm( {
+			...this.props.signupForm,
+			timezone: maybeRewriteTimezone( timezone ),
```

Comment:
I think `maybeRewriteTimezone()` is not needed here since `setTimezone()` is called only with the dropdown selected value?

Reply:
I was unsure how to test it and unsure about the logic, I'll remove it. Thanks!